docs/standards/rpc-api.md

```diff
@@ -10,14 +10,109 @@ sidebar_position: 10
 
-The [LUKSO Extension](../guides/browser-extension/install-browser-extension.md) uses new RPC API which are described here.
+## Supported Methods
 
-## Methods
+<table>
+  <tr>
+    <td>Name</td>
+    <td>Type</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sign">eth_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#personal_sign">personal_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_getAccounts">eth_getAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_requestAccounts">eth_requestAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sendTransaction">eth_sendTransaction</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#wallet_switchEthereumChain">wallet_switchEthereumChain</a></td>
+    <td>standard</td>
+  </tr>
+   <tr>
+    <td><a href="#up_addTransactionRelayer">up_addTransactionRelayer</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_import">up_import</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_generateLsp23Address">up_generateLsp23Address</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+</table>
 
-### up_addTransactionRelayer
+## Signing
 
-Add a custom relayer.
+### eth_sign
+
+**_NOTE:_** We encourage developers to use eth_sign for signing purposes.
+
+While a security issue potentially existed in the initial implementation on Ethereum, the current implementation has no such potential exploit. As such usage of this method is preferable to [personal_sign](#personal_sign).
+
+This method returns a [EIP-191](https://eips.ethereum.org/EIPS/eip-191) signature over the data provided to the call.
+It requests that the user provides an Ethereum address that should sign the transaction as well as the data (encoded bytes) that are to be executed.
+
+Returns
+`string[]` - on a successful call the method returns a signature, a string representing hex encoded bytes or 4001 - if the user rejects the requets
+
+### personal_sign
+
+The personal_sign endpoint is enabled to allow for backward compatibility. However, its use is not recommended.
+Some libraries such as Ethers.js end up using personal_sign under the hood. That is why, for compatibility reasons, personal_sign is left enabled. Note however that personal_sign ultimately acts as a proxy, redirecting the call to the [eth_sign](#eth_sign) method.
+
+## Standard
+
+### eth_sendTransaction {#eth_sendTransaction}
+
+Creates new message call transaction or a contract creation, if the data field contains code, and signs it using the account specified in `from`. This method requires that the user has granted permission to interact with their account first, so make sure to call eth_requestAccounts first.
```

Comment:
> or a contract creation, if the data field contains code

It's not only the data field that highlights a transaction as "contract creation" transaction, e.g. `to` address should be empty when it's a deployment of a contract. 

I'd leave out the "contract creation" part from this documentation.

Also, I think that's important to mention, that the transaction will not be signed by the UP itself but by the controller address that has enough permissions to act on said UP. 

And I'm not sure what you meant by making sure the user granted permission and making "sure to call eth_requestAccounts first.". Even though `eth_requestAccounts` could be informative it doesn't influence granted permissions. Happy to discuss 🙂